dags/process_elife_zip_dag.py

```diff
@@ -90,8 +95,38 @@ def prepare_jats_xml_for_libero(**context):
 
     s3 = get_aws_connection('s3')
     response = s3.get_object(Bucket=DESTINATION_BUCKET, Key=xml_path)
-    xml = etree.parse(BytesIO(response['Body'].read()))
-    return etree.tostring(xml)
+    article_xml = etree.parse(BytesIO(response['Body'].read()))
+
+    # get article id
+    article_id = article_xml.xpath(
+        '/article/front/article-meta/article-id[@pub-id-type="publisher-id"]'
```

Comment:
the XPath queries may benefit from being extracted as string constants, I do this with [CSS queries](https://github.com/elifesciences/elife-spectrum/blob/master/spectrum/xpub.py#L25-L27) for example

Reply:
I would agree if the paths were used multiple places but at the moment it's only for single use.